webapp/components/_new/features/SearchResults/SearchResults.vue

```diff
@@ -315,7 +355,7 @@ export default {
       variables() {
         const { firstPosts, postsOffset, search } = this
         return {
-          query: search,
+          query: search.replace(/^([!@#&])/, ''),
```

Comment:
That makes an error in the webapp tests:
`TypeError: Cannot read property 'replace' of undefined`